robot-tests/test-cases/onos-config.robot

```diff
@@ -0,0 +1,101 @@
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
+# SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+*** Settings ***
+Documentation             Smoke test for onos-config
+Suite Setup               Setup Suite
+Suite Teardown            Teardown Suite
+Library                   String
+Library                   Process
+Resource                  ../libraries/onos.robot
+
+*** Variables ***
```

Comment:
These are instantiated twice. They will also be imported when onos.robot gets imported.